### ISSUE TITLE
[email templates] Revise `images` partial

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#11678]
The `images` partial includes the App Store and Google Play badges, and it passes the `productName` variable for image alt text. As there are both FxA and SubPlat emails that either should have `Firefox` set to `productName` or do not have a `productName` variable (reminder emails to finish account setup, payment emails, etc), a conditional needs to be added so the templates do not break. While this affects all templates, the following emails are currently not loading within Storybook:

* subscriptionAccountReminderFirst
* subscriptionAccountReminderSecond
* subscriptionsPaymentExpired
* subscriptionsPaymentProviderCancelled

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXA-4462)

**Comment by data-sync-user** [comment]
➤ Bianca Oltean commented:

This issue was verified as fixed. The mentioned templates are loading in Storybook.

!templates in Storybook.gif|width=1905,height=1012!
